pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -457,9 +457,10 @@ type NetworkPolicyNodeStatus struct {
 
 // NetworkPolicyEvaluation contains the request and response for a NetworkPolicy evaluation.
 type NetworkPolicyEvaluation struct {
-	metav1.TypeMeta `json:",inline"`
-	Request         *NetworkPolicyEvaluationRequest  `json:"request,omitempty" protobuf:"bytes,1,opt,name=request"`
-	Response        *NetworkPolicyEvaluationResponse `json:"response,omitempty" protobuf:"bytes,2,opt,name=response"`
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,3,opt,name=metadata"`
```

Comment:
Could we add a comment inline why this is 3 to avoid it to be “corrected” by accident?

Could this be fixed when we introduce a new version for NetworkPolicyEvaluation API? I suppose yes.

Reply:
Done.

> Could this be fixed when we introduce a new version for NetworkPolicyEvaluation API? I suppose yes.

Yes I also believe that this can be fixed in a future version, without any negative impact. Protobuf message types are completely independent across versions.